pom.xml

```diff
@@ -0,0 +1,356 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ibmix.magkit</groupId>
+    <artifactId>magkit-pom</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>IBM iX Magkit-POM</name>
+    <description>IBM iX Magkit POM which should be used as the top level pom of the project structure for iX-Magkit projects. This POM
+        defines the common management sections with their default settings for dependencies and plugins.
+    </description>
+    <url>https://github.com/IBM/magkit-pom</url>
+    <inceptionYear>2013</inceptionYear>
+
+    <organization>
+        <name>IBM iX</name>
+        <url>https://ibmix.de</url>
+    </organization>
+
+    <properties>
+        <!-- Magnolia version, override this to use another version -->
+        <magnoliaVersion>6.2.33</magnoliaVersion>
+        <magnolia.content-types.version>1.2</magnolia.content-types.version>
+        <magnolia.site.version>1.4</magnolia.site.version>
+
+        <source.javaversion>11</source.javaversion>
+        <target.javaversion>11</target.javaversion>
+
+        <source.encoding>UTF-8</source.encoding>
+        <project.build.sourceEncoding>${source.encoding}</project.build.sourceEncoding>
+
+        <!-- checkstyle can be disabled with this property -->
+        <checkstyle.skip>false</checkstyle.skip>
+        <checkstyle.config>src/config</checkstyle.config>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
```

Comment:
The Magnolia module dependencies are very specific for the vanity url module. Perhaps use a Magnolia bom dependency in that place instead of these specific modules.

Reply:
done!

Reply:
```suggestion
            <dependency>
            <!-- Importing depMgt -->
            <dependency>
                <groupId>info.magnolia.bundle</groupId>
                <artifactId>magnolia-bundle-parent</artifactId>
                <version>${magnoliaVersion}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
             <!-- Import depMgt for 3rd-party libraries (BOM) -->
            <dependency>
                <groupId>info.magnolia.boms</groupId>
                <artifactId>magnolia-external-dependencies</artifactId>
                <version>${magnoliaVersion}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```

Reply:
What do u mean here? I removed magnolia-bundle-parent and magnolia-external-dependencies